modules/rostests/apitests/ntdll/NtQueryInformationToken.c

```diff
@@ -156,7 +156,7 @@ QueryTokenPrivilegesTests(
                                      BufferLength,
                                      &BufferLength);
     ok_ntstatus(Status, STATUS_SUCCESS);
-    ok(Privileges->PrivilegeCount == 20, "The number of privileges must be 20 (current number %lu)!\n", Privileges->PrivilegeCount);
+    ok(Privileges->PrivilegeCount == 21, "The number of privileges must be 21 (current number %lu)!\n", Privileges->PrivilegeCount);
```

Comment:
I don't know what version of Windows 2003 you're using but these tests do actually pass on Windows Server 2003 SP2 (using a local administrator account).
![Capture](https://github.com/reactos/reactos/assets/34916900/6d5f411a-d9c6-4783-9b40-3a06f1eb77eb)
If they fail on WHS but subsequently work on a newer version with latest service pack, then that's the WHS fault and you cannot simply change the test values like that to support a broken behavior .

To make WHS at least happy for the sake of it, you can adjust the failed tests to return two different results like what you do [here](https://github.com/reactos/reactos/pull/6791/files#diff-8522089fe4d960adedb6883ea0fde20e9d651c81d6914da9b3c430d83a532ec1R228).

Reply:
I removed this one for now. Might do a better fix later, unless you do it first :)